frontend/src/routes/[[lang=locale]]/(voters)/questions/+layout.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import {getTopBarProgressContext} from '../../topBarProgress.context.js';
+
+  export let data;
+
+  $: data.opinionQuestions.then((qs) => setProgressBarMax(qs.length + 1));
```

Comment:
NB. We need to use the total number of questions in the selected categories for the max value.

Reply:
The difference is that currently we use the sum of the questions in the categories the user has selected for answering (on `/questions/+page.svelte`) instead of all questions. The list of selected questions is calculated on `/[questionId]/+page.svelte: ~87`:

```ts
  $: Promise.all([$opinionQuestions, $opinionQuestionCategories]).then(
    ([qq]) => (questions = filterAndSortQuestions(qq, $firstQuestionId, $selectedCategories))
  );
```

And `questions.length` used for the total.